easybuild/easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.7-iomkl-2018a-R-3.5.0.eb

```diff
@@ -0,0 +1,937 @@
+        'checksums': ['5e8b32d65f77a86d17d90fd8690fc085aa0612df8018e4d6d6c1a60fa65776e4'],
+    }),
+    ('snow', '0.4-2', {
+        'checksums': ['ee070187aea3607c9ca6235399b3db3e181348692405d038e962e06aefccabd7'],
+    }),
+    ('BiocParallel', '1.14.1', {
+        'checksums': ['d6c8c76de48fa0f3e2e81abac0a3e39b205719080917aea4bf18272ade0c0f03'],
+    }),
+    ('Rsamtools', '1.32.0', {
+        'checksums': ['c6aaf17a5d895c78642020cf92204128654319ac533c49f5471d346279f1c5e8'],
+    }),
+    ('DelayedArray', '0.6.1', {
+        'checksums': ['bafd11ce39175b9ca62dfa8dec81310cb2049f21b1c426a2907383d8fad75c6a'],
+    }),
+    ('lattice', '0.20-35', {
+        'checksums': ['0829ab0f4dec55aac6a73bc3411af68441ddb1b5b078d680a7c2643abeaa965d'],
+    }),
+    ('Matrix', '1.2-14', {
+        'checksums': ['49a6403547b66675cb44c1afb04bb87130c054510cb2b94971435a826ab41396'],
+    }),
+    ('SummarizedExperiment', '1.10.1', {
+        'checksums': ['81dacf10280d5f6e79a5eed84c1ccaac7cf536a47f5b569e4a676bcc12ed7f6c'],
+    }),
+    ('GenomicAlignments', '1.16.0', {
+        'checksums': ['5c31ea7deaf17c3ea3a66fe41dfce754b5495f65a28be7860042ae392d13f102'],
+    }),
+    ('hwriter', '1.3.2', {
+        'checksums': ['6b3531d2e7a239be9d6e3a1aa3256b2745eb68aa0bdffd2076d36552d0d7322b'],
+    }),
+    ('RColorBrewer', '1.1-2', {
+        'checksums': ['f3e9781e84e114b7a88eb099825936cc5ae7276bbba5af94d35adb1b3ea2ccdd'],
+    }),
+    ('latticeExtra', '0.6-28', {
+        'checksums': ['780695323dfadac108fb27000011c734e2927b1e0f069f247d65d27994c67ec2'],
+    }),
+    ('ShortRead', '1.38.0', {
+        'checksums': ['cb058f83d68937dc1ec1d6662d2597c65bbdfc943a24ffe5bbab7085cf1f1f0d'],
+    }),
+    ('graph', '1.58.0', {
+        'checksums': ['a99c54df8616226cba72b277e9e8f45ffd6f017010ae56a2d1e199316921a4ff'],
+    }),
+    ('affyio', '1.50.0', {
+        'checksums': ['56885f62402c8d68acca2f17fbdb3cd6af2356c50bb26c6cc1d779c8ace5056a'],
+    }),
+    ('BiocInstaller', '1.30.0', {
+        'checksums': ['e0395bb6b4d81f7629658730a350bf924d92ab7d414419cea56e1707437ce1f5'],
+    }),
+    ('preprocessCore', '1.42.0', {
+        'checksums': ['7371a739d3481544c2e95e956d873fcc03e8f03e4761fd573d3b95747ec8caa9'],
+    }),
+    ('affy', '1.58.0', {
+        'checksums': ['26cef02431581909471bb637ba03c736fde6787699593c9c0ea2bea8cb41b86b'],
+    }),
+    ('GO.db', '3.6.0', {
+        'checksums': ['120d1485f17a118b4d3145f40e5b26dbe6b70cc6093bec382defaa94b9c7223a'],
+    }),
+    ('limma', '3.36.2', {
+        'checksums': ['c4f80de2f2233c58fc658548293f8cdec3dfe76bda0e27457ce284df3dfff196'],
+    }),
+    ('RBGL', '1.56.0', {
+        'checksums': ['0e5a056a1a63e711bf73ec9978b525dfb7adb49f63e01cba2bd15f5cab384942'],
+    }),
+    ('org.Hs.eg.db', '3.6.0', {
+        'checksums': ['27c838a6d343e9cb869d20dacf489fc0e3eac7a881a8bc2840a3a0ad10235b49'],
+    }),
+    ('XML', '3.98-1.11', {
+        'checksums': ['505bd948152213c22e255b8d38b303c0fac5e113c0f9b1bcc9bfa62e1edd3fa2'],
```

Comment:
checksum failed, I'm getting `90c83af4b1a304d04f97eaf0c5b651b02135dc15cce7e4e38d732d48ae9e30b8` on a fresh download...

Reply:
Deleted my existing download and ran inject-checksums